challenges/headstorm-fec-jehillert/README - Applicant's Notes.txt

```diff
@@ -0,0 +1,136 @@
+-------------------------------------------------------------------------------
```

Comment:
I love all the documentation you provided! It's great to get some ideas of why you made the decisions you made with this (admittedly vague) challenge.